app/cypress/e2e/0-ndr-core-tests/privacy_page.cy.js

```diff
@@ -0,0 +1,62 @@
+const { Roles } = require('../../support/roles');
+
+describe('Privacy Page', () => {
+    const baseUrl = Cypress.config('baseUrl');
+    const startUrl = '/';
+    const privacypolicyUrl = '/privacy-policy';
+    const rolesToTest = [Roles.GP_ADMIN, Roles.GP_CLINICAL, Roles.PCSE];
+
+    const loginAndVisitPrivacyPolicyPage = (role) => {
+        cy.login(role);
+        cy.visit(privacypolicyUrl);
+    };
+
+    it('verify privacy policy link in start up page', { tags: 'regression' }, () => {
+        cy.visit(startUrl);
+        cy.url().should('eq', baseUrl + startUrl);
+        cy.get('.nhsuk-footer__list-item-link')
```

Comment:
Is this check needed if we're checking the main link below directly with the data id?